pcsx2/gui/Panels/GameFixesPanel.cpp

```diff
@@ -107,6 +107,10 @@ Panels::GameFixesPanel::GameFixesPanel( wxWindow* parent )
 		{
 			_("VU I bit Hack avoid constant recompilation (Scarface The World Is Yours)"),
 			wxEmptyString
+		},
+		{
+			_("Set EE Cycle rate to 100% when a FMV plays"),
```

Comment:
Clarification should be added here that the EE cyclerate will reset to its normal speed (100% isn't very clear)

The notable cases where it is needed should be noted as well, such as R&C.